core/src/main/java/org/jivesoftware/sparkimpl/certificates/CertificateController.java

```diff
@@ -127,20 +168,48 @@ private void fillCertTableWithKeyStoreContent(String storePath) {
 				String alias = (String) store.nextElement();
 				X509Certificate certificate = (X509Certificate) trustStore.getCertificate(alias);
 				CertificateModel certModel = new CertificateModel(certificate, alias);
-				if (storePath.equals(EXCEPTIONS)) {
-					certModel.setExempted(true);
-				}
 				certificates.add(certModel);
 			}
-
 		} catch (KeyStoreException | NoSuchAlgorithmException | CertificateException | IOException e) {
 			Log.warning("Cannot access Truststore, it might be not set up", e);
 		}
 
 	}
-	
+/**
+ * Return file which contains certificate with given alias;
+ * @param alias
+ * @return
+ */
+	public File getAliasKeyStore(String alias) {
```

Comment:
To some extend I need to use alias as deleteEntry/setCertificateEntry methods of the KeyStore work only with certificate alias. I am not sure if changing it for signature would help a lot, especially as it is now in use only for deleting certificates and to firstly get certificates I already had to do  

`X509Certificate certificate = (X509Certificate) keyStore.getCertificate(alias);`

Still I remember that for sure using unique values is important to identify certificates for creating SSLContext/TrustManager.